core/templates/services/assets-backend-api.service.ts

```diff
@@ -160,6 +161,13 @@ export class AssetsBackendApiService {
       entityType, entityId, filename, AppConstants.ASSET_TYPE_THUMBNAIL);
   }
 
+  getBlobDataFromBlobUrl$(url: string): Observable<Blob> {
```

Comment:
This should be an async function? Any specific reason for `$` at the end of the function name here?

Reply:
`$` is like the async prefix we have but for Observables. It is a widely convention in angular and RxJS based codebases. This not an sync function because this doesn't return a promise.

Reply:
Thanks for the info.